src/styles/blocks/card.scss

```
@@ -0,0 +1,100 @@
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__title {
+    color: $color-main-accent;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 0 4px;
```

reset margin for `h3` similar you did for `p` and use only `margin-bottom` here
```suggestion
    margin-bottom: 4px;
```